staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/schema/structural.go

```diff
@@ -104,29 +104,30 @@ type Extensions struct {
 	// x-kubernetes-list-type annotates a list to further describe its topology.
 	// This extension must only be used on lists and may have 3 possible values:
 	//
-	// 1) `atomic`: the list is treated as a single entity, like a scalar.
+	// 1) `` for atomic: the list is treated as a single entity, like a scalar.
 	//      Atomic lists will be entirely replaced when updated. This extension
 	//      may be used on any type of list (struct, scalar, ...).
 	// 2) `set`:
 	//      Sets are lists that must not have multiple items with the same value. Each
-	//      value must be a scalar (or another atomic type).
+	//      value must be a scalar, an object with x-kubernetes-map-type `atomic` or an
+	//      array with x-kubernetes-list-type `atomic`.
 	// 3) `map`:
 	//      These lists are like maps in that their elements have a non-index key
 	//      used to identify them. Order is preserved upon merge. The map tag
 	//      must only be used on a list with elements of type object.
-	XListType *string
+	XListType string
 
 	// x-kubernetes-map-type annotates an object to further describe its topology.
 	// This extension must only be used when type is object and may have 2 possible values:
 	//
-	// 1) `granular`:
+	// 1) `` for granular:
```

Comment:
I don't understand this change

Reply:
empty string if it is granular (the default for maps). In other words: we only set this value for the non-standard cases.